lessons/06-html-layout/index.html

```diff
@@ -0,0 +1,35 @@
+<html>
+    <title>Jaynen's Blog</title>
+    <header>
+        <h1>Jaynen's Blog</h1>
+    </header>
+    <main>
+        <article>
+            <a href="posts/post-1.html">Important News Article 1</a>
+            <p>
+                <em>Posted on <span title="2019-01-06 08:15:36">Jan 06, 2019</span></em>
+            </p>
+            <p>
+                Lorem, ipsum dolor sit amet consectetur adipisicing elit. Ea quod beatae repudiandae inventore minus dignissimos repellendus sint, vero id explicabo. Aspernatur ex rerum mollitia asperiores maxime unde vitae obcaecati praesentium?
+            </p>
+        </article>
+        <article>
+            <a href="posts/post-2.html">Important News Article 2</a>
+            <p>
+                <em>Posted on <span title="2019-03-21 08:15:36">March 21, 2019</span></em>
+            </p>
+            <p>
+                Lorem, ipsum dolor sit amet consectetur adipisicing elit. Ea quod beatae repudiandae inventore minus dignissimos repellendus sint, vero id explicabo. Aspernatur ex rerum mollitia asperiores maxime unde vitae obcaecati praesentium?
+            </p>
+        </article>
+        <article>
+            <a href="posts/post-3.html">Important News Article 3</a>
+            <p>
+                <em>Posted on <span title="2019-04-18 08:15:36">April 18, 2019</span></em>
+            </p>
+            <p>
+                Lorem, ipsum dolor sit amet consectetur adipisicing elit. Ea quod beatae repudiandae inventore minus dignissimos repellendus sint, vero id explicabo. Aspernatur ex rerum mollitia asperiores maxime unde vitae obcaecati praesentium?
+            </p>
+        </article>
+    </main>
```

Comment:
This looks good! It just needs to be surrounded by a `<body>` tag

Every single one of your html files should look like:

```html
<html lang="en">
<head>
  <title>Document</title>
</head>
<body>
  ... your content here
</body>
</html>
```